libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineApiTest.kt

```diff
@@ -456,7 +456,7 @@ class MapboxRouteLineApiTest {
 
             api.updateUpcomingRoutePointIndex(routeProgress)
 
-            verify { mockVanishingRouteLine.primaryRouteRemainingDistancesIndex = 6 }
+            verify { mockVanishingRouteLine.primaryRouteRemainingDistancesIndex = 7 }
```

Comment:
Why has this changed?

Reply:
It's an artificial test that checks a weird case then the index returned by NN is not valid (see the test description). Before part of `allRemainingPoints` was calculated using turf and did not rely on this index. Now it does rely on it and thus we get a different result. I found it OK to change the expected value because either way with an invalid index we get some weird result. The point is that we don't crash.